be/src/storage/lake/lake_persistent_index.h

```diff
@@ -130,6 +146,9 @@ class LakePersistentIndex : public PersistentIndex {
     static void pick_sstables_for_merge(const PersistentIndexSstableMetaPB& sstable_meta,
                                         std::vector<PersistentIndexSstablePB>* sstables, bool* merge_base_level);
 
+    // Check if this rowset need to rebuild, return `True` means need to rebuild this rowset.
+    static bool rowset_rebuild_checker(const RowsetMetadataPB& rowset, uint32_t rebuild_rss_id);
```

Comment:
checker is na noun, we need use a verb in function name.
should be `needs_rowset_rebuild`?

Reply:
done